src/common/gui/main-styles.ts

```diff
@@ -2311,9 +2311,24 @@ styles.registerStyle("main", () => {
 			padding: "35%",
 			margin: "-35% -35%",
 		},
-		".color-picker": {
-			height: px(30),
-			width: px(100),
+		".custom-color-container .text-field": {
+			"padding-top": "0px",
+		},
+		".custom-color-container .text.input": {
+			"text-transform": "uppercase",
+			width: "9ch",
+		},
+		".custom-color-container .inputWrapper:before": {
+			content: '"#"',
+			color: theme.content_message_bg,
+		},
+		".custom-color-container .color-option": {
```

Comment:
these two could be specified directly in code, right?

Reply:
Yes, it could it just has to be conditional